### DISCUSSION TITLE
How to install pnpm in AWS CodeBuild?

### DISCUSSION THREAD

**Discussion by bestickley** [pnpm/pnpm#5448] [Q&A]
How can I install pnpm in AWS CodeBuild? I keep getting an error that the command pnpm is not recognized.

**Comment by bestickley** [comment]
I ended up with:
```bash
curl -fsSL https://get.pnpm.io/install.sh | SHELL=`which bash` bash -
. ~/.bashrc
# below lines I thought were needed, but they're actually not
# export PNPM_HOME="/root/.local/share/pnpm"
# export PATH="$PNPM_HOME:$PATH"
```
I hope there is a better way :)

**Reply by nick-kang** [reply]
The above is the best way if you want the latest pnpm version. 

This is an alternative, simpler way if you're on Node v16+. However, it doesn't give you the latest pnpm version.
```bash
corepack enable
corepack prepare pnpm@latest --activate
```

**Reply by bestickley** [reply]
Got it. Thanks for the alternative answer, @nick-kang!
